src/theme/Navbar/styles.css

```
@@ -0,0 +1,7 @@
+.navbar__item--dock {
+  display: flex;
+}
+
+.navbar__item--dock > * + * {
```

That would be a little clearer for me but the result is the same

```suggestion
.navbar__item--dock > * {
```

---

This breaks the toggle button for me in Chrome because it shifts the icons, but not the background. I'm going to change it to use margins btw, because it seems semantically more correct to create space between adjacent siblings.